### PR TITLE
Add transparency information to social preview docs

### LINKED ISSUE #16417: Add transparency information to social preview documentation
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Customizing your repository's social media preview](https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview)

### What part(s) of the article would you like to see updated?

Github social preview supports using transparent PNGs, but this does not seem to be documented. Adding a small subsection about transparency would be welcome. Usage of transparent PNGs can be helpful for branding as many platforms support dark/light themes and transparent PNGs can work well with multiple themes.

### Additional information

You can read my general GitHub feedback here https://github.com/github/feedback/discussions/13109#discussion-3951682, for more information.

### PR DESCRIPTION
**Pull request by aisgbnok**
### Why:

Closes #16417

### What's being changed:

Addition of a small "Transparency" section to the ["Customizing your repository's social media preview"](https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) page.

Here is what the built and rendered addition looks like:

image

If you have any feedback, questions, concerns, etc. don't hesitate to reply. I can add more information. For example, here is what a social preview with a transparent PNG looks like in Google Chat/Spaces.

social-preview-transparency-chat

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@aisgbnok 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by Aradsaeed** [comment]
> ### Why:
> Closes #16417
> 
> ### What's being changed:
> Addition of a small "Transparency" section to the ["Customizing your repository's social media preview"](https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) page.
> 
> Here is what the built and rendered addition looks like:
> 
> image
> 
> If you have any feedback, questions, concerns, etc. don't hesitate to reply. I can add more information. For example, here is what a social preview with a transparent PNG looks like in Google Chat/Spaces.
> 
> social-preview-transparency-chat
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md
Comment (mchammer01) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md:35]: Slightly rewording + avoiding passive voice + putting the advice given in the last sentence in its own paragraph to highlight it more.

```suggestion
We support PNG images with transparency. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this may not always be the case. 

When using an image with transparency, keep in mind how it may look on different color backgrounds or platforms that don't support transparency.
```

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md
Comment (mchammer01) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md:38]: Making this tip consistent with our other tips
```suggestion
**Tip:** If you aren't sure, we recommend using an image with a solid background.
```

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md
Comment (mchammer01) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md:33]: As per our content model
```suggestion
## About transparency
```

**Review by mchammer01 — COMMENTED** [review]
@aisgbnok 👋🏻 - thank you for your contribution ✨ 
I've made some suggestions (most of them relating to our content model) that I'm going to apply.
I'll merge this once I've checked that the preview works fine 🚢

**Comment by mchammer01** [comment]
@ramyaparimi 👋🏻 - I think this is ready to be merged. The deployment is behaving oddly and I can't see the **View deployment** button, nor the updates I've just made on the preview. I raised this yesterday with the team in charge but haven't heard anything back.

**Comment by aisgbnok** [comment]
@mchammer01 Hey, the [preview link seems to be updated for me?](https://docs-16418-4d4234.preview.ghdocs.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) 
image

I like your changes, writing documentation can be hard. It takes a certain mindset. My only concern is that `**Tip**` doesn't look right? Thanks! ☺️

**Comment by mchammer01** [comment]
@aisgbnok - yes, thanks, I noticed that the preview eventually got updated but things are a bit slow today 😟 
I'll see if I can fix the rendering on the tip 🙂

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md
Comment (mchammer01) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md:41]:
```suggestion
{% tip %}

```

**Comment by mchammer01** [comment]
I fixed the other tip too and hopefully we should be good to go. Thanks for highlighting the rendering issue @aisgbnok ✨ 
Just waiting for an updates preview.

**Comment by mchammer01** [comment]
I've marked this as `ready-to-merge` but one of the tests is failing. Re-running it.

**Comment by ramyaparimi** [comment]
@mchammer01 Since [this test](https://github.com/github/docs/runs/5834865471?check_suite_focus=true) is still failing, I will wait until the test is passing to merge this PR 💖. I am not entirely sure why the test is failing though, is there anything I can do to fix the error?

**Comment by ramyaparimi** [comment]
@aisgbnok That is what I thought as well that it could be the size of the image or a format error. But seems like it is something related to an input. I will ask my team about the error and get back to you 💖.

**Comment by heiskr** [comment]
Looks like we can't write to forks using GitHub Actions. I've disabled the optimize images for the public repo. If possible to optimize the image before we merge that would be great, but disabling for the public repo should be enough to get the build green.

**Comment by ramyaparimi** [comment]
@mchammer01 I will wait for your one final approval before I can get this merged 💖.

**Review by mchammer01 — APPROVED** [review]
Final approval, as requested 😄

**Comment by aisgbnok** [comment]
@heiskr I tried to run optimize images from my repo, but it skipped. I think because https://github.com/aisgbnok/docs/blob/db6301e9381d28fb1576cd4ac606d01f878529c3/.github/workflows/optimize-images.yml#L19

Not sure if there is another way, or if I should just leave it be? Thanks for the help!

**Comment by heiskr** [comment]
No worries, we can always run it later 🤷🏼

**Pull request merged**
